src/components/UniversalNavbar/UniversalNavbar.js

```diff
@@ -86,87 +83,74 @@ class UniversalNavbar extends React.Component {
           }}
         >
           <Layout.ScrollDetector>
-            <Media.PhoneOnly>
-              {/* Hamburger button */}
-              <div style={{ zIndex: 2, position: 'fixed', top: 24, right: 24 }}>
-                <TransformingBurgerButton
-                  showMobileMenu={showMobileMenu}
-                  clickHandler={this.toggleHamburger}
-                />
-              </div>
+            <div className={'UniversalNavbar--hamburger'}>
+              <TransformingBurgerButton
+                showMobileMenu={showMobileMenu}
+                clickHandler={this.toggleHamburger}
+              />
+            </div>
+
+            <div className={'UniversalNavbar--phoneOnly'}>
+              {/* Dark green mobile menu, shows up when hamburger is clicked */}
               <div
                 className={
-                  'universal-navbar-root' +
-                  ' ' +
-                  'universal-navbar-mobileRoot' +
-                  ' flex items-center justify-between'
+                  showMobileMenu ? 'shownMobileMenu' : 'hiddenMobileMenu'
                 }
               >
-                {/* Dark green mobile menu, shows up when hamburger is clicked*/}
-                <div
-                  className={
-                    showMobileMenu ? 'shownMobileMenu' : 'hiddenMobileMenu'
-                  }
-                >
-                  <a href="/">
-                    {LogoWhite({ className: 'universal-navbar-logo' })}
-                  </a>
-                  <Spacer.H56 />
-                  {navbarLinks.map((l) => (
-                    <div key={l.title + 'mobile'} style={{ marginBottom: 24 }}>
-                      <TitleXLarge.Sans.Regular400>
-                        <a key={l.title + 'mobile'} href={l.href}>
-                          {l.title}
-                        </a>
-                      </TitleXLarge.Sans.Regular400>
-                    </div>
-                  ))}
-                  <div style={{ position: 'absolute', bottom: 40 }}>
-                    <a href="/term" onClick={this.props.trackCtaClick}>
-                      <Button.Medium.BlackOutline>
-                        Check my price
-                      </Button.Medium.BlackOutline>
-                    </a>
+                <a href={LINKS.INDEX.href}>
+                  {LogoWhite({ className: 'universal-navbar-logo' })}
+                </a>
+                <Spacer.H56 />
+                {LINKS.NAVLINKS.map((l) => (
+                  <div key={l.title + 'mobile'} style={{ marginBottom: 24 }}>
+                    <TitleXLarge.Sans.Regular400>
+                      <a key={l.title + 'mobile'} href={l.href}>
```

Comment:
UUID instead maybe?

Reply:
Sure, though I didn't write this, or even modify it in this PR really. I see this PR as scoped to solving the bug, which is the hamburger button webkit glitch due to the Media helpers. All this diff is just indentation mostly.

Reply:
Also maybe @iamvictorli just did this? Saw a similar diff a minute ago.

Reply:
Hrm, yeah, I may have said the same thing on his PR haha. Let's call this non blocking. I think we may want to work on using uuid across places at some point but if there are no errors in console we have bigger things to fix as a priority. Just know that uuid is available also in CMS now if you'd like to use it.